.github/workflows/deploy-gh-pages-explorer.yml

```diff
@@ -28,24 +28,27 @@ jobs:
       url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Check out Git repository
```

Comment:
I think it's nice if we always describe what a step does, makes the logs look prettier ✨